src/pdf/struct.js

```diff
@@ -10,20 +11,24 @@ var dpmm = 72 / 25.4; // dots per mm at 72dpi
  * @param {number} options.y
  * @param {number} options.width
  * @param {number} options.height
+ * @param {number} [options.pageHeight] Defaults to A4
  * @param {() => void} fn
  */
 export function structFigure(doc, options, fn) {
     var x = options.x * dpmm;
     var y = options.y * dpmm;
     var width = options.width * dpmm;
-    var height = options.width * dpmm;
-    var bbox = [x, y, x + width, y + height];
+    var height = options.height * dpmm;
+    var pageHeight = (options.pageHeight || defaultPageHeight) * dpmm;
+    var bbox = [x, pageHeight - y - height, x + width, pageHeight - y];
```

Comment:
lets say
`var bbox = [x1, y1, x2, y2]`
then in de above when we assume `height` and `width` are positive numbers
then in the above code that means:
`x2 >= x1` and `y2 >= y1`
so to me this still feels like it is clockwise, `[left, top, right, bottom].`

could it be that `y` is negative ?

Reply:
`y2 >= y1` because the bounding box origin is at the bottom of the page.